.travis.yml

```diff
@@ -5,10 +5,12 @@ go:
     - 1.7
 dist: trusty
 sudo: required
+env:
+  - TAGS='!ostree'
```

Comment:
Shouldn't just not having TAGS defined make this work?

Reply:
yes true, I'll drop that